type/__debconf_set_selections/gencode-remote

```diff
@@ -51,4 +51,15 @@ then
 		{
 			printf "set %s %s %s %s\n", $1, $2, $3, $4
 		}' "${filename}" >>"${__messages_out:?}"
+
+	if test -f "${__object:?}/parameter/reconfigure"
+	then
+		package="$(awk '{print $1}' "${filename}")"
```

Comment:
🤔 what if a user updates debconf entries for more than one package at once?

Is there a way to figure out which packages would need to be reconfigured?

Reply:
Then they will use `require=""` and `__check_messages`?

Reply:
Or we can loop over...

Reply:
A while loop can help here.